<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAudioNotificationComponent</class>
 <widget class="QFrame" name="CAudioNotificationComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>297</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notification</string>
  </property>
  <layout class="QVBoxLayout" name="vl_AudioComponent">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="sa_AudioComponent">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="sa_AudioComponentInner">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>285</width>
        <height>304</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="12" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioPTTBlocked">
         <property name="text">
          <string>PTT blocking</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QCheckBox" name="cb_SetupAfvClicked">
         <property name="text">
          <string>AFV click</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_Notifications">
         <property name="toolTip">
          <string>Notifications</string>
         </property>
         <property name="text">
          <string>Volume:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="sb_NotificationValueVolume">
         <property name="suffix">
          <string> volume 25-100 (notifications)</string>
         </property>
         <property name="minimum">
          <number>25</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="wi_SoundDir" native="true">
         <layout class="QHBoxLayout" name="hl_SoundDir" stretch="0,0,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="le_SoundDir">
            <property name="placeholderText">
             <string>your private sound directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_SoundReset">
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="pb_SoundDir">
            <property name="text">
             <string>[...]</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioNotificationTextMessageFrequency">
         <property name="text">
          <string>Text message on ATC frequency</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioNotificationTextMessageUnicom">
         <property name="text">
          <string>Text message on UNICOM</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioNotificationTextCallsignMentioned">
         <property name="text">
          <string>Text message mentions my callsign</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="cb_SetupAfvBlocked">
         <property name="text">
          <string>AFV blocking</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioNotificationTextMessagePrivate">
         <property name="text">
          <string>Private text message</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_SoundDirectory">
         <property name="text">
          <string>Dir.:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioNotificationAtcTunedIn">
         <property name="text">
          <string>ATC frequency selected</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioPTTClickDown">
         <property name="text">
          <string>PTT click (key down)</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioNotificationTextMessageSupervisor">
         <property name="text">
          <string>Supervisor text message</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioPTTClickUp">
         <property name="text">
          <string>PTT click (key up)</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QCheckBox" name="cb_SetupAudioNotificationAtcTunedOut">
         <property name="text">
          <string>ATC frequency deselected</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sa_AudioComponent</tabstop>
  <tabstop>le_SoundDir</tabstop>
  <tabstop>pb_SoundReset</tabstop>
  <tabstop>pb_SoundDir</tabstop>
  <tabstop>sb_NotificationValueVolume</tabstop>
  <tabstop>cb_SetupAudioNotificationTextMessageFrequency</tabstop>
  <tabstop>cb_SetupAudioNotificationTextMessageUnicom</tabstop>
  <tabstop>cb_SetupAudioNotificationTextCallsignMentioned</tabstop>
  <tabstop>cb_SetupAudioNotificationTextMessagePrivate</tabstop>
  <tabstop>cb_SetupAudioNotificationTextMessageSupervisor</tabstop>
  <tabstop>cb_SetupAudioPTTClickDown</tabstop>
  <tabstop>cb_SetupAudioPTTClickUp</tabstop>
  <tabstop>cb_SetupAudioPTTBlocked</tabstop>
  <tabstop>cb_SetupAfvClicked</tabstop>
  <tabstop>cb_SetupAfvBlocked</tabstop>
  <tabstop>cb_SetupAudioNotificationAtcTunedIn</tabstop>
  <tabstop>cb_SetupAudioNotificationAtcTunedOut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
